umcontroller/umcontroller.go

```diff
@@ -225,10 +247,16 @@ func New(config *config.Config, storage storage, certProvider CertificateProvide
 	umCtrl *Controller, err error,
 ) {
 	umCtrl = &Controller{
-		storage:              storage,
+		config:        config,
+		storage:       storage,
+		certProvider:  certProvider,
+		cryptocontext: cryptocontext,
+		insecure:      insecure,
+
 		eventChannel:         make(chan umCtrlInternalMsg),
 		nodeInfoProvider:     nodeInfoProvider,
 		nodeInfoChannel:      nodeInfoProvider.SubscribeNodeInfoChange(),
+		certChan:             make(<-chan *iamanager.CertInfo),
```

Comment:
remove make

Reply:
initializes certChannel for server.handleChannels() goroutine for insecure case